README.md

```diff
@@ -91,7 +102,17 @@ Please be aware that the native `source.yml` connection set up in the package wi
 
 To connect your multiple schema/database sources to the package models, follow the steps outlined in the [Union Data Defined Sources Configuration](https://github.com/fivetran/dbt_fivetran_utils/tree/releases/v0.4.latest#union_data-source) section of the Fivetran Utils documentation for the union_data macro. This will ensure a proper configuration and correct visualization of connections in the DAG.
 
-## (Optional) Step 4: Additional configurations
+## (Optional) Step 4: Populating the User Models
```

Comment:
Yeah I made it its own section initially because it involves 3 models which I felt was substantial, but you make a good point in that the variable isn't necessary for the entirety to run. I consolidated this to the additional configs section!